buildspec.yml

```diff
@@ -0,0 +1,35 @@
+version: 0.2
+
+eb_codebuild_settings:
+  CodeBuildServiceRole: arn:aws:iam::697877139013:role/service-role/code-build-nessie-service-role
+  ComputeType: BUILD_GENERAL1_MEDIUM
+  Image: aws/codebuild/nodejs:6.3.1
```

Comment:
BOAC uses `nodejs:6.3.1` image due because gulp is run during build. For Nessie, the image reqt might change.